tools/doc/markdown.js

```diff
@@ -2,18 +2,29 @@
 
 const visit = require('unist-util-visit');
 
+const referenceToLocalMdFile = /^(?![+a-z]+:)([^#?]+)\.md(#.+)?$/i;
+
 module.exports = {
-  replaceLinks
+  replaceLinks,
+  referenceToLocalMdFile,
 };
 
 function replaceLinks({ filename, linksMapper }) {
   return (tree) => {
     const fileHtmlUrls = linksMapper[filename];
 
+    visit(tree, (node) => {
+      if (node.url) {
```

Comment:
I would like to know what this data structure looks like.

Reply:
Here's the result of a `console.log(node)` here:

```js
{
  type: 'link',
  title: null,
  url: '#cli_diagnostic_dir_directory',
  children: [ { type: 'text', value: '--diagnostic-dir', position: [Position] } ],
  position: Position {
    start: { line: 131, column: 1, offset: 3405 },
    end: { line: 131, column: 50, offset: 3454 },
    indent: []
  }
}
```